bark_slidebot_config:
  pick_jpose: [0, -1.509143666630127, 1.9175460974322718, -1.9791599712767542, -1.5711048285113733, -3.141593]  # joint variablesy
  photo_jpose: [0, -1.3847, 2.3688, -4.1254, -1.5314, 0]  # joint variables
  object_in_world_frame: []
  camera_in_tcp_frame: [0, 0.039, -0.08, -1.5708, 0, 0]
  tcp_in_robot_eef_frame: [0, 0, 0.148, 0, 0, 0]